src/commands/math.ts

```diff
@@ -42,131 +47,147 @@ var MathElement = P(Node, function(_, super_) {
   };
   // Remove nodes that are more than `cutoff`
   // blocks deep from this node.
-  _.removeNodesDeeperThan = function (cutoff) {
+  removeNodesDeeperThan (cutoff:number) {
     var depth = 0;
-    var queue = [[this, depth]];
-    var current;
+    var queue:[[MQNode, number]] = [[this, depth]];
+    var current:[MQNode, number] | undefined;
 
     // Do a breadth-first search of this node's descendants
     // down to cutoff, removing anything deeper.
-    while (queue.length) {
-      current = queue.shift();
-      current[0].children().each(function (child) {
+    while (current = queue.shift()) {
+      var c = current;
```

Comment:
Its not a big deal wither way, but @mikehaverstock was there a reason I'm missing for why we need this new variable c?